app/controllers/portal.js

```diff
@@ -149,6 +76,98 @@ export default Ember.Controller.extend({
      */
     formValidation (validation) {
       this.set('isFormInvalid', validation.errors.length !== 0)
+    },
+    updateDOM: function () {
+      Ember.$('#username_label').text(this.get('username'))
+      Ember.$('#token_label').text(this.get('token'))
+      Ember.$('.general_info').show()
+      Ember.$('.action_generate').show()
+      if (this.get('token') === '~') {
+        Ember.$('.restricted_info').show()
+        Ember.$('.action_unrestrict').show()
+        Ember.$('.action_restrict').hide()
+      } else {
+        Ember.$('.restricted_info').hide()
+        Ember.$('.action_restrict').show()
+        Ember.$('.action_unrestrict').hide()
+      }
+    },
+    createUser: function (element) {
+      return this.get('store').createRecord('developer', {
+        username: element.label,
+        token: element.value
+      })
+      .save()
+      .then((res) => {
+        console.log('For the user with this username: ' + res.get('username') + ', their testing token is: ' + res.get('token'))
```

Comment:
It goes on the screen in the function above. It is not necessary to log the statement and it will be deleted.